pynest/nest/__init__.py

```diff
@@ -93,11 +93,17 @@ def __init__(self, name):
         _rel_import_star(self, ".lib.hl_api_types")
 
         # Lazy loaded modules. They are descriptors, so add them to the type object
-        type(self).spatial = _lazy_module_property("spatial")
         type(self).raster_plot = _lazy_module_property("raster_plot")
+        type(self).spatial = _lazy_module_property("spatial")
```

Comment:
Is there a reason to change this order?

Reply:
Sorry. I thought that I could sort them alphabetically. But I undo this changes.

Reply:
You can :) I didn't see you put them alphabetically :D